mvebu64boot.c

```diff
@@ -741,7 +743,7 @@ static bool loop_boot_pattern(int fd)
 		return false;
 	}
 
-	printf("BootROM is ready for image file transfer\n");
+	fprintf(stderr, "BootROM is ready for image file transfer\n");
```

Comment:
This change is obviously wrong. Message is informative and not error, so it must **not** be printed to stderr.

Reply:
So this filter is obviously broken and cannot work. mvebu64boot communicates directly with tty UART device and not some random stdin or stdout. Fix your filter instead.